### ISSUE TITLE
not isomorphic

### ISSUE THREAD

**Issue by samthomson** [xiaoyunyang/isomorphic-router-demo#6]
I'm not sure the app is rendering on the server as expected. The home `/` route serves the SPA which when loaded in the client calls the server for content.

**Comment by latchy** [comment]
You're right, this isn't isomorphic.. you can tell by looking at the page source.

**Comment by samthomson** [comment]
:man_shrugging: c'est la vie, starting playing with next.js instead, that is really cool and very extensible.
